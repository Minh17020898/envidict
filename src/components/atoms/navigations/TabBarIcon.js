import React from 'react'
import { Ionicons } from '@expo/vector-icons'

const TabBarIcon = (props) => {
  return (
    <Ionicons
      name={props.name}
      size={30}
      style={{ marginBottom: -3 }}
      color={props.focused ? '#000000' : '#d3d3d3'}
    />
  )
}

export default TabBarIcon
