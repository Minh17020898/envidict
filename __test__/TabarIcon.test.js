import 'react-native'
import React from 'react'

import renderer from 'react-test-renderer'

import TabBarIcon from '../src/components/atoms/navigations/TabBarIcon'

describe('Loading navigation icons', () => {
  it(`renders icon heart`, () => {
    const tree = renderer.create(<TabBarIcon name="md-heart" focused={true} />).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it(`renders icon home`, () => {
    const tree = renderer.create(<TabBarIcon name="md-home" focused={false} />).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it(`renders icon book`, () => {
    const tree = renderer.create(<TabBarIcon name="md-book" focused={false} />).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it(`renders icon settings`, () => {
    const tree = renderer
      .create(<TabBarIcon name="md-settings" focused={false} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
