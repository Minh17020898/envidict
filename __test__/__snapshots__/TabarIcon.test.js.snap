// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders icon book 1`] = `<Text />`;

exports[`App renders icon heart 1`] = `<Text />`;

exports[`App renders icon home 1`] = `<Text />`;

exports[`App renders icon settings 1`] = `<Text />`;

exports[`App renders the loading screen 1`] = `<Text />`;

exports[`Loading navigation icons renders icon book 1`] = `<Text />`;

exports[`Loading navigation icons renders icon heart 1`] = `<Text />`;

exports[`Loading navigation icons renders icon home 1`] = `<Text />`;

exports[`Loading navigation icons renders icon settings 1`] = `<Text />`;
